packages/jest-util/src/fake_timers.js

```diff
@@ -225,11 +225,37 @@ export default class FakeTimers {
     }
 
     if (i === this._maxLoops) {
+      const timers = Object.assign({}, this._timers);
```

Comment:
you don't need to do this if you only do `Object.keys` on it (just don't mutate the object)